.vscode/tasks.json

```diff
@@ -2,24 +2,17 @@
 	"version": "2.0.0",
 	"tasks": [
 		{
-			"label": "build",
-			"command": "dotnet",
-			"type": "shell",
-			"args": [
-				"build",
-        "--nologo",
-				"--property:GenerateFullPaths=true",
-				"--consoleloggerparameters:NoSummary"
-			],
-			"group": {
-				"kind": "build",
-				"isDefault": true
-			},
-			"presentation": {
-				"reveal": "silent"
-			},
-			"problemMatcher": "$msCompile"
-		},
+            "label": "build",
+            "command": "dotnet",
+            "type": "process",
+            "args": [
+                "build",
+                "${workspaceFolder}/Test/NodeApi.Test.csproj",
+                "/property:GenerateFullPaths=true",
+                "/consoleloggerparameters:NoSummary"
+            ],
+            "problemMatcher": "$msCompile"
```

Comment:
Keep the `group` and `presentation` sections deleted above. The `group` section sets this as the default build task so that it is run by the Ctrl+Shift+B shortcut. The `presentation` section prevents the build console window from being shown unless there are errors.

Reply:
The original tasks.json is restored.